airnode-rrp-log/src/main.rs

```diff
@@ -26,15 +26,19 @@ impl State {
 }
 
 impl reader::EventHandler for State {
-    fn on(&mut self, l: web3::types::Log) -> () {
+    fn on(&mut self, l: web3::types::Log, pretty_print: bool) -> () {
```

Comment:
Could you please move `pretty_print` out of the handler into `State` properties which are set in its constructor? Everything else LGFM